YourToDo/templates/auth/registration.html

```diff
@@ -2,10 +2,46 @@
 
 {% block content %}
 	<div class="container">
-		<h1>Register</h1>
-		<form action="/registration/" method="post">{% csrf_token %}
-			{{form}}
-			<input type="submit" value="Register" />
-		</form>
+		<div class="form-registration">
+			<h2>Registration</h2>
+			<form action="/registration/" method="post">{% csrf_token %}
+				<div class="username">
+					<label for="{{ form.username.id_for_label }}">Username:</label>
+					{{ form.username }}
+					<p>Length: 30 characters or fewer. Letters, digits and '
+                    '@/./+/-/_ only.'<p>
+                    {{ form.username.errors }}
+				</div>
+				<div class="first_name">
```

Comment:
Each input field here is contained by a div, which is a good idea. Instead of attaching unused classes to all of these, we might want to attach a class that will set the width of the label to a consistent amount for a better design.